Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:20-alpine AS deps
```

Comment:
Pls koristi zadnji LTS (node 22). Node 20 ide u maintainance mode za koji tjedan.

Preporučam napraviti nešto kao:
```dockerfile
FROM node:22-alpine AS base


FROM base AS deps
...
```